res/xml/file_paths.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <paths xmlns:android="http://schemas.android.com/apk/res/android">
-    <files-path path="pictures" name="pictures/"/>
+    <files-path name="pictures" path="pictures/"/>
```

Comment:
This is a file for paths constants?

Reply:
Yes, you need to whitelist those explicitly since Nougat, because they changed it so that you couldn't expose URIs with `file://` outside of the app, you need to get a `content://` uri in one of those paths.